common/switchboard.hpp

```diff
@@ -68,7 +72,7 @@ const record_header __switchboard_topic_stop_header {"switchboard_topic_stop", {
  * switchboard::reader<topic1_type> topic1 = switchboard.get_reader<topic1_type>("topic1");
  *
  * // Get a writer on topic2
- * switchboard::writer<topic2_type> topic2 = switchboard.get_reader<topic2_type>("topic1");
+ * switchboard::writer<topic2_type> topic2 = switchboard.get_writer<topic2_type>("topic1");
```

Comment:
The name of the topic should be topic2. `switchboard.get_writer<topic2_type>("topic2");`